.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -6,15 +6,19 @@ labels: ".Needs Triage, Type:Bug"
 assignees: ''
 
 ---
+**Read this before creating a bug report**
+- in case any of the sections is incomplete, we might close the bug report as we won't be able to reproduce it
+- have you checked if the bug has already been reported? as we close duplicate reports, please search on the issue tracker if the problem has been reported before
```

Comment:
```suggestion
- have you checked if the bug has already been reported? As we close duplicate reports, please search on the issue tracker if the problem has been reported before
```

Reply:
Was this meant to sound like - search **because** we will close if it is a duplicate report?

`As we close duplicate reports` is confusing me a bit.